rllib/algorithms/appo/torch/appo_torch_rl_module.py

```diff
@@ -17,6 +17,10 @@
 class APPOTorchRLModule(PPOTorchRLModule, RLModuleWithTargetNetworksInterface):
     @override(PPOTorchRLModule)
     def setup(self):
+        # APPO is asynchronous and therefore all modules must be learner modules.
```

Comment:
Good question. I think that the following happens: When synching with the learner the learner was giving back inference module weights and the modules in the runner were erroring out. I think it might have to do with the  logic in the APPO module having a target.